source/dub/packagemanager.d

```diff
@@ -780,15 +719,178 @@ private enum LocalOverridesFilename = "local-overrides.json";
 
 
 private struct Repository {
-	Path path;
 	Path packagePath;
 	Path[] searchPath;
 	Package[] localPackages;
 	PackageOverride[] overrides;
 
 	this(Path path)
 	{
-		this.path = path;
-		this.packagePath = path ~"packages/";
+		this.packagePath = path;
+	}
```

Comment:
The decision to use "packages/" as a subdir for holding downloaded packages is now localised in `Dub.init()` (which should definitely be renamed, something for another day...). With that done there doesn't seem to be any point in `Repository.path`.